aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -32,99 +34,96 @@ static void parallel_apply(Tensor& result, const Tensor& self, F f) {
 
   static tbb::affinity_partitioner ap;
```

Comment:
Is it safe to use the same `affinity_partitioner` from multiple threads at the same time?